dcpy/lifecycle/ingest/configure.py

```diff
@@ -67,6 +68,13 @@ def get_version(source: Source, timestamp: datetime | None = None) -> str:
             return extract_socrata.get_version(source)
         case GisDataset():
             return publishing.get_latest_gis_dataset_version(source.name)
+        case DEPublished():
```

Comment:
We only have one of these for now. There's some discussion to be had for how we want to do this exactly - ideally, it's coming from `package` folders I think. But I think this is an issue worth punting - again, we currently only get ztl straight from our own output folders